DisplayBundle/Resources/public/requireConf.js

```diff
@@ -2,6 +2,7 @@ require.config({
     baseUrl: '/bundles',
     paths: {
         openOrchestraCss: 'openorchestradisplay/libs/openOrchestraCss',
-        jquery: 'openorchestradisplay/libs/jquery-2.0.2.min'
+        jquery: 'openorchestradisplay/libs/jquery-2.0.2.min',
+        bootstrap: 'openorchestradisplay/libs/bootstrap/bootstrap.min'
```

Comment:
yes it is required by the behavior chosen for the carousel block : http://www.jssor.com/bootstrap/bootstrap-slider.html